tests/random_schema.cc

```diff
@@ -263,6 +263,28 @@ std::vector<data_value> generate_frozen_tuple_values(std::mt19937& engine, value
     return values;
 }
 
+data_model::mutation_description::collection generate_user_value(std::mt19937& engine, const user_type_impl& type,
+        value_generator& val_gen) {
+    using md = data_model::mutation_description;
+
+    auto key_dist = std::uniform_int_distribution<uint16_t>(0, type.size() - 1);
+
+    std::map<uint16_t, md::atomic_value> collection;
+    for (size_t i = 0; i < type.size(); ++i) {
```

Comment:
Instead of type.size() we should generate a number in the range [min_valid_value, type.size()), where min_valid_value is 1, as I suppose empty mutations don't make sense (do they?).

Reply:
Read my answer to your comment below.

Reply:
What I wanted is to have UDT mutations with only some of the fields set. I guess you already achieve this to some extent by generating colliding keys (below). Still smaller UDT mutations that have just one or two fields set will be very unlikely to be generated.

Reply:
Ok, maybe it'll be better to make the number of non-null fields uniformly distributed. I'll change this.

Reply:
Done